packages/aws-cdk-lib/aws-ses/lib/configuration-set.ts

```diff
@@ -73,6 +73,32 @@ export interface ConfigurationSetProps {
    * @default - use the default awstrack.me domain
    */
   readonly customTrackingRedirectDomain?: string;
+
+  /**
+   * The Virtual Deliverability Manager (VDM) options that apply to the configuration set
+   *
+   * @default - use account level settings
+   */
+  readonly vdmOptions?: VdmOptions;
+}
+
+/**
+ * Properties for the Virtual Deliverability Manager (VDM) options that apply to the configuration set
+ */
+export interface VdmOptions {
+  /**
+   * Whether engagement metrics are enabled for the configuration set
```

Comment:
I find it more explicit to say exactly what the `true` value is. This way if the variable is something like `isFeatureDisabled`, you know exactly what `true` means
```suggestion
   * If true, engagement metrics are enabled for the configuration set
```